test/parallel/xyk-pallet.API.mintLiquidity.test.ts

```diff
@@ -218,13 +230,29 @@ describe("xyk-pallet - Mint liquidity tests: with minting you can", () => {
       poolBalanceWhenCreated[1].add(secondCurrencyAmountLost),
     ]).collectionBnEqual(poolBalanceAfterMinting);
 
-    testUser1.validateWalletReduced(firstCurrency, injectedValue);
-    testUser1.validateWalletReduced(secondCurrency, secondCurrencyAmountLost);
+    let diffFromWallet = testUser1
+      .getAsset(secondCurrency)
+      ?.amountBefore!.sub(injectedValue);
+    expect(testUser1.getAsset(firstCurrency)?.amountAfter!).bnEqual(
+      diffFromWallet!
+    );
+
```

Comment:
Hey @eddyjlsh ,
I can see that you removed all usage for the validators.  I created those validators to avoid repeating and do maths operations in the tests so they are easy to read. 
I think that removing the validators is a bit overkill :). 
Looking at all the changes you have done, I would consider the following.
Instead of

`testUser.validateWalletReduce( tokenId, value )` 

What if we create  something like:

`Validators.ValidateWalletReduced( testUser1, tokenId, value)`

So :
```
class Validator{
    ValidateWalletReduced( user, assetId, amount) {
        //implement almost the same thing as before but retrieving the assetvalues from user with GetAsset.
   }
}

```
If you do not like the above solution, we can go directly with this PR as it is. 
Just thinking on encapsulate the wallet handling and maths outside of the test itself.

Reply:
Ping me when you read this, so I either continue with the review, either wait for the new strategy for this decoupling.